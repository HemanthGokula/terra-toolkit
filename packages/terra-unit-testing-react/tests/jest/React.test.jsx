import React from 'react';

describe('Sanity React Jest Test', () => {
  it('Correctly tests a snapshot', () => {
    const div = shallow(<div>Test</div>);
    expect(div).toMatchSnapshot();
  });
});
