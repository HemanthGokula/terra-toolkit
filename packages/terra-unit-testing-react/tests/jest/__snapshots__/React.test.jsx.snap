// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sanity React Jest Test Correctly tests a snapshot 1`] = `
<div>
  Test
</div>
`;
